### DISCUSSION TITLE
Train. Multi GPU

### DISCUSSION THREAD

**Discussion by Oleg-1978** [ultralytics/ultralytics#2645] [Docs]
Pease write instruction how to run correctly Yolo8 on multi GPU. 
Instruction Yolo5 do not work.

**Comment by glenn-jocher** [comment]
@Oleg-1978 hello! To train YOLOv8 on multiple GPUs, you'll want to ensure that you have a compatible environment set up with all necessary dependencies, including PyTorch with CUDA support for GPU acceleration.

Once your environment is ready, you can simply use the `train` command with YOLOv8. The training script will automatically detect and use all available GPUs. There's no need for any special flags or commands; the multi-GPU support is built-in and should work out of the box.

If you encounter any issues or have specific questions about your setup, please provide details about the error messages or behavior you're observing, and we'll do our best to assist you further. Happy training! 🚀
